janis_core/tool/tool.py

```diff
@@ -265,6 +271,47 @@ def tests(self) -> Optional[List[TTestCase]]:
         """
         return None
 
+    def minimal_test(self):
```

Comment:
Let's add some comment here to help people want to use this to understand what it does.